### PR TITLE
RTMD-75 - wait for translations to load before booting app

### PR DESCRIPTION
**Pull request by joefitter**
* move app init into i18n on ready callback

### CODE REVIEW

**Comment by JoeChapman** [comment]
Out of interest, does all of that code need to be inside the `ready` event callback?

**Comment by joefitter** [comment]
not if we use bootstrap...

**Comment by JoeChapman** [comment]
Ok, so we aren't using bootstrap in this, and we don't need to either, so I suppose the anser to the original question is "yes"? @joefitter

**Comment by joefitter** [comment]
yes and no actually. If we are looking up the translations and using them outside of the middleware  as per here - https://github.com/UKHomeOffice/gro/blob/master/app.js#L127-L129 then yes we need to wait. If the translate function is passed to a middleware which uses it internally, it seems the translations have already loaded by this point.

**Comment by joefitter** [comment]
I am not sure if this is just an asyc fluke, or if this will always be the case - if the latter then no, we can move it all outside of the on ready callback except for the terms and cookies middlewares

**Comment by joefitter** [comment]
TLDR; I have no idea.

**Comment by joefitter** [comment]
derp
